app/index.html

```diff
@@ -4,28 +4,51 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
     <title>Boats Animator</title>
-    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/fonts.css">
+    <link rel="stylesheet" href="css/animator.css">
 </head>
 
-<body class="index" onload="canDisplayNews()">
-    <header class="app-info">
-        <h1>Boats Animator</h1>
-        <footer>Copyright (C) Charlie Lee 2015 <span style="float: right; text-align: right; padding: 0 4%;"><a href="#" onclick="gui.Shell.openExternal('http://charlielee.uk')">charlielee.uk</a></span></footer>
+<body class="animator" onload="canDisplayNews()">
```

Comment:
Again, this class on the body was to separate the two stylesheets. However, neither `.animator` or `.index` is used, so you can just remove this class attribute.